recipes-core/logrotate/files/logrotate-frequent.cron

```diff
@@ -0,0 +1 @@
+*/10 * * * * root /usr/sbin/logrotate /etc/logrotate.conf
```

Comment:
We install the config to `${D}${sysconfdir}/logrotate.d/default-logrotate` but here refer to `/etc/logrotate.conf`.

Reply:
this is the main lorotate.conf that exists and it includes the logrotate.d directory in it